### PR TITLE
feat: add yield version for Mean Finance

### PR DESCRIPTION
**Pull request by 0xsambugs**
This PR is doing quite a few things:
1. We are adding a new version of Mean that will be released soon. We call it the "yield" version, since it will enable yield-while-dca
1. We are now querying the tokens in the last version from our new API, instead of using the subgraph
1. We are also removing the subgraph dependencies form the previous versions (that still hold quite a lot of tvl since some of them are still in use). We are making this change so that we can remove all external dependencies, since we won't be adding new tokens to the "legacy" versions
1. We are also using `multicall` now, since it was taking quite a lot of time to test the changes 😅 
1. We are removing arbitrum from the chains list, since we never actually enabled it on the UI. We will re-add it when we deploy officially

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/meanfinance: 
        
 
  
    
    
    Error: TVL for key optimism is not a number, instead it is NaN
    Truncated error stack:
    
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:82:11)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:167:9
        at async Promise.all (index 0)
        at async Promise.all (index 1)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:212:3

File: projects/meanfinance/index.js [hunk 1]
```diff
@@ -105,13 +86,12 @@ module.exports = {
...
-  arbitrum: getV2TvlObject('arbitrum'),
    hallmarks: [
     [1638850958, "V2 Beta launch on Optimism"],
     [1643602958, "V2 full launch"],
     [1646367758, "Deployment on Polygon"],
     [1650082958, "Protocol is paused because a non-critical vulnerability"],
     [1653366158, "V2 Relaunch"],
-    [1654057358, "OP launch bring more users into Optimism and benefit Mean"]
+    [1654057358, "OP launch bring more users into Optimism and benefits Mean"]
   ]
```
Comment (0xsambugs) [inline projects/meanfinance/index.js:96]: I have a small question. Can we add hallmarks for the future? If so, how would it look in the UI. We were thinking of adding easter eggs to announce the launch of yield-while-dca, and we though that DefiLlama could be a cool place to add one

**Comment by 0xsambugs** [comment]
> Error while running adapter at projects/meanfinance:
> 
> ```
> Error: TVL for key optimism is not a number, instead it is NaN
> Truncated error stack:
> 
>     at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:82:11)
>     at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:167:9
>     at async Promise.all (index 0)
>     at async Promise.all (index 1)
>     at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:212:3
> ```

That's weird. When I run `node test.js projects/meanfinance/index.js` locally it works correctly

**Comment by llamatester** [comment]
The adapter at projects/meanfinance exports TVL: 
        
 
 
    polygon                   185.28 k
    optimism                  394.23 k
    ethereum                  28.41 k
    
    total                    607.91 k

**Pull request merged**